### DISCUSSION TITLE
LQI, map and routers

### DISCUSSION THREAD

**Discussion by choeflake** [Koenkk/zigbee2mqtt#6694] [General]
Hi,
I have some questions about the technical background of Zigbee networks.

### Which LQI is shown in Zigbee2Mqtt?
For example I have 3 bulbs in one fixture. The LQI's of the 3 bulbs are: 15, 36 and 102. Physically there are also other bulbs between this fixture and the coordinator.
I would expect that when 1 bulb has an LQI of 102, the other 2 will use that bulb to communicate to the coordinator (mesh network).
Showing the map, I see that the bulb of 15 is connected to a bulb in between with an LQI of 87. That bulb is connected to the coordinator with an LQI of 48/42 (shown in the map, what means 48/42?)

### How to read the map
When I create a map, I see single devices and groups of devices without a line to the coordinator. When I enable 'isASibling', a lot (really a lot) of extra lines (dotted and solid) become visible.
What means 'isParent', 'isAChild', 'isASibling' in terms of a Zigbee network (or map).

### Bulbs, repeaters and DBm
Since a few versions ago of Zigbee2mqtt, we can set the DBm of the coordinator. In home situation with a house full of bulbs (routers) and other powered devices, what is exactly the advantage? I would say that a low transmission to the nearest routers should also work pretty well. Since every bulb has some other bulbs at short distance. This would create a balanced mesh network. In stead of a high value of devices communicating directly with the coordinator. And keeps the total radiation low. (Note, the more hops, the higher the latency I think.)
So my question is, why has Zigbee2MQTT the setting for DBm and what about dedicated repeaters (like Tradfri) and/or bulbs etc (routers).

The background of my questions comes from the fact that some of my devices regularly experience timeouts or very slow reactions. I have read almost all basic topics about range etc. I use channel 25, my coordinator is a LAUNCHXL-CC26X2R1.

**Comment by Koenkk** [comment]
> Which LQI is shown in Zigbee2Mqtt?

The linkquality of the last hop is shown. E.g. coordinator -> bulb a -> bulb b, in this case the linkquality for coordinator -> bulb a is shown.

> How to read the map

isASibling lines are shown when 2 devices are connected through the same parent. This does not matter in Zigbee, only parent <-> child matters


> Bulbs, repeaters and DBm

This depends a lot on the network setup. The dBm shouldn't be too high, but not too low either, the optimal dBm highly depends on the setup. I expect 5 dBm to be a fine value for most setups, but e.g. 20 dBm for the CC1352P seems overkill.

**Reply by choeflake** [reply]
> > Which LQI is shown in Zigbee2Mqtt? 

> The linkquality of the last hop is shown. E.g. coordinator -> bulb a -> bulb b, in this case the linkquality for coordinator -> bulb a is shown.

I don't know if the data is available, but the lowest LQI in the chain seems te be more 'accurate' to me? 

> > How to read the map 

> isASibling lines are shown when 2 devices are connected through the same parent. This does not matter in Zigbee, only parent <-> child matters

Ok, I understand. But in my map I see groups of devices (and also single devices) which do not have a line to the coordinator. That line is shown when I enable 'isASibling', I cannot match that...

> > Bulbs, repeaters and DBm

> This depends a lot on the network setup. The dBm shouldn't be too high, but not too low either, the optimal dBm highly depends on the setup. I expect 5 dBm to be a fine value for most setups, but e.g. 20 dBm for the CC1352P seems overkill.

I understand that it depends on the setup. On the other side, the most networks don't differ a lot. One ore more bulbs is every room and one or more end devices. So, every bulb has an another bulb within max 4-5 meters, separated by a wall or floor.

**Reply by Koenkk** [reply]
> I don't know if the data is available, but the lowest LQI in the chain seems te be more 'accurate' to me?

This data isn't available.

> Ok, I understand. But in my map I see groups of devices (and also single devices) which do not have a line to the coordinator. That line is shown when I enable 'isASibling', I cannot match that...

@nurikk can you explain this?

**Comment by sjorge** [comment]
Maps in the frontend doesn't look correct for me either with everything always disconnected with a few exceptions, I stopped using it and went back to generating the manually, if I include routes I usually get nice connections to all my devices from the coordinator via 1 or more routers to all end devices or other routers.
